erica/application/JobService/job_service.py

```diff
@@ -54,18 +49,12 @@ def add_to_queue(self, payload_dto: BaseDto, client_identifier: str, job_type: R
         created = self.repository.create(request_entity)
         logging.getLogger().info(f"EricaRequest created with id: {created.request_id}")
 
-        job = self.background_worker.enqueue(
-            self.job_method,
-            created.request_id,
-            ttl=get_settings().ttl_queuing_job_in_sec,
-            retry=Retry(max=get_settings().queue_retry_repetitions,
-                        interval=get_settings().queue_retry_interval_seconds)
-        )
-        logging.getLogger().info(f"Job created with id {job.id} for EricaRequest with id {created.request_id}")
+        self.job_method(created.request_id)
+        logging.getLogger().info(f"Job created with id for EricaRequest with id {created.request_id}")
```

Comment:
So I see this correctly that there'll be no retries?

Reply:
In that case we should probably also delete the whole ERIC_ERRORS_WITH_RETRY part?